browser_history/cli.py

```diff
@@ -115,12 +118,12 @@ def main():
                 args.browser,
             )
             sys.exit(1)
-
-        if args.type == "history":
-            fetch_map[args.type]["var"] = browser_class().fetch_history()
-        elif args.type == "bookmarks":
-            fetch_map[args.type]["var"] = browser_class().fetch_bookmarks()
-
+        fetch_map = {
+            "history": {"var": h_outputs, "fun": browser_class().fetch_history},
+            "bookmarks": {"var": b_outputs, "fun": browser_class().fetch_bookmarks},
+            "cookies": {"var": c_outputs, "fun": browser_class().fetch_cookies},
+        }
+        fetch_map[args.type]["var"] = fetch_map[args.type]["fun"]()
```

Comment:
Is the `fun` used anywhere else?

Reply:
Nope Its used in the `all` condition only at the start that's why didn't think it would be bad to reinitialise it

Reply:
I looked at `fetch_map` again and it looks like the `var` key isn't required. We could just replace `fetch_map[args.type]["var"]` with one variable, say `outputs`.
I'll open an issue for this (it's better not to do it in this PR since this will go into a separate branch).

Reply:
I think I made this for the `all` feature in bookmarks which I removed later, might have missed noticing this xD